pkg/test/framework/components/istio/operator.go

```diff
@@ -241,33 +242,21 @@ func deploy(ctx resource.Context, env *kube.Environment, cfg Config) (Instance,
 	}
 
 	// install control plane clusters (can be external or primary)
-	errG := multierror.Group{}
 	for _, cluster := range env.KubeClusters {
 		if env.IsControlPlaneCluster(cluster) {
 			cluster := cluster
-			errG.Go(func() error {
-				return installControlPlaneCluster(i, cfg, cluster, istioctlConfigFiles.iopFile)
-			})
+			if err = installControlPlaneCluster(i, cfg, cluster, istioctlConfigFiles.iopFile); err != nil {
```

Comment:
Why have we gotten rid of the concurrency here? IIUC this was done intentionally to decrease startup time.

FYI @stevenctl

Reply:
the concurrency is added recently by @stevenctl ,  Let me consume it here to keep the concurrency.